packages/docs/docs/bots/bots-in-production.md

```diff
@@ -94,11 +94,34 @@ ls dist
 
 ```
 
-## Linking your code to a Bot Resource
+## Creating your Bot 
 
-The next step will be to create a [`Bot` resource](/docs/api/fhir/medplum/bot) using the Medplum App and link your code to the resource.
+Next step is to create the bot. 
 
-First, we'll have to create the resource in the Medplum App. If you haven't already done so, follow the instructions on the [Bot Basics tutorial](./bot-basics/#creating-a-bot) to create a new Bot resource.
+Navigate to the [Project Admin panel](https://app.medplum.com/admin/project) and copy the ID of your project. That will be your `project-id`. 
+
+Taking the `source-file` we just created at `src/my-first-bot.ts`, we will use the `create-bot` command. In our example 
+
+```bash
+npx medplum create-bot <bot-name> <project-id> <source-file> <dist-file>
+```
+
+Running this command does the following:
+
+1. Creates the Bot resource 
+2. Creates a ProjectMembership resource that connects it to a project
+3. Saves the bot to the associated project in the Medplum database
+4. Adds a bot entry to the `medplum.config.json` file in the `bots` array
+
+:::caution Note
+If you see an error, try running the command again. If it fails after 3 tries, please [**submit a bug report**](https://github.com/medplum/medplum/issues/new) or [**contact us on Discord**](https://discord.gg/UBAWwvrVeN)
+:::
```

Comment:
After this, add an instruction for the users to _open_ the medplum.config.json, just so they know what it does. Copy the table from line 144-149